decoupled-site/frontend/codegen.yaml

```diff
@@ -1,4 +1,4 @@
-schema: https://cg.optimizely.com/content/v2?experimental=true&auth=INPUT_SINGLE_KEY_HERE
+schema: https://dev.cg.optimizely.com/content/v2?experimental=true&auth=4vnxJuIFeaCP5IRaMcIExDKtK3nDjOv2aLCYONXxScqP0lL5
```

Comment:
Could you remove creds in there? As a suggestion, we can guide users to create `.env.local` file and configure our codegen config to use it .

`DOTENV_CONFIG_PATH=./.env.local graphql-codegen --config codegen.yml -r dotenv/config`

https://stackoverflow.com/questions/69461989/is-there-any-way-to-use-next-env-in-codegen-yaml

Reply:
@cimdalli I updated